menu/sotm_2021.json

```diff
@@ -0,0 +1,24 @@
+{
+        "version": 2021070100,
+        "url": "https://sotm.osmz.ru/2021.xml",
+        "title": "State of the Map 2021",
+        "start": "2021-07-09",
+        "end": "2021-07-11",
+        "metadata": {
+                "icon": "https://2021.stateofthemap.org/img/sotm_2021-logo-small.png",
+                "links": [
+                        {
+                                "url": "https://2021.stateofthemap.org/",
+                                "title": "Website"
+                        }
+                ],
+                "rooms": [
```

Comment:
This is not necessary by the way if you're not specifying any metadata. It's all in the schedule file. Listing rooms in the JSON is useful if you want to specify locations for them, etc. Want to remove it again? (I should add this to the test..)

Reply:
Thanks for the suggestion, I used sotm-2019 as a template, when we had rooms with actual locations :see_no_evil: